tests/virtualization/universal/storage.pm

```diff
@@ -35,7 +35,7 @@ sub run_test {
     ## Basic Dir Storage Management
     my $dir_vol_size = '100M';
     my $dir_vol_resize = '200M';
-    virt_storage_management($dir_pool_name, size => $dir_vol_size, dir => 1, resize => $dir_vol_resize);
+    virt_storage_management($dir_pool_name, \@guests, size => $dir_vol_size, dir => 1, resize => $dir_vol_resize);
```

Comment:
It's reference to array. Perl put all sub parameters in a array @_. If passing a array and other  parameters, we usually pass that array reference in case the array elements mix up with parameters